test/e2e_node/remote/remote.go

```diff
@@ -207,22 +207,31 @@ func getTestArtifacts(host, testDir string) error {
 	if err := os.MkdirAll(logPath, 0755); err != nil {
 		return fmt.Errorf("failed to create log directory %q: %w", logPath, err)
 	}
-	// Copy logs to artifacts/hostname
-	if _, err := runSSHCommand(host, "scp", "-r", fmt.Sprintf("%s:%s/results/*.log", GetHostnameOrIP(host), testDir), logPath); err != nil {
-		return err
+	// Copy logs (if any) to artifacts/hostname
```

Comment:
Check if there are things in `results/*.log` before running scp (just like we do below)